packages/mui-material/src/Slide/Slide.js

```diff
@@ -84,16 +84,17 @@ const defaultEasing = {
   exit: easing.sharp,
```

Comment:
Shouldn't easing be also taken from the theme?

Reply:
Good point Michal! Thanks for calling that out as well

Reply:
You are right!